src/messaging/tests/MessagingContext.h

```diff
@@ -59,13 +61,19 @@ class MessagingContext : public IOContext
 
     SecureSessionMgr & GetSecureSessionManager() { return mSecureSessionMgr; }
     Messaging::ExchangeManager & GetExchangeManager() { return mExchangeManager; }
+    mcsp::MessageCounterManager & GetMessageCounterManager() { return mMessageCounterManager; }
+
+    SecureSessionHandle GetSessionToLocal();
+    SecureSessionHandle GetSessionToPeer();
 
     Messaging::ExchangeContext * NewExchangeToPeer(Messaging::ExchangeDelegate * delegate);
     Messaging::ExchangeContext * NewExchangeToLocal(Messaging::ExchangeDelegate * delegate);
 
 private:
     SecureSessionMgr mSecureSessionMgr;
     Messaging::ExchangeManager mExchangeManager;
+    mcsp::MessageCounterManager mMessageCounterManager;
```

Comment:
Are these ever used at the same time?
Maybe tests should be responsible for allocating these.

Reply:
done